SlackPirate.py

```diff
@@ -459,6 +461,62 @@ def find_private_keys(token, output_info: ScanningContext):
     print(termcolor.colored("\n"))
 
 
+def find_pinned_messages(token, output_info: ScanningContext):
+    """
+    This function looks for pinned messages across all Slack channels the token has access to - including private
+    channels. We often find interesting information in pinned messages.
+    The function first calls the conversations.list API to grab all Slack channels from the Workspace. It then passes
+    the channel_id to pins.list which will either return pinned messages or not. If it does, dump to file :-)
+    """
+
+    print(termcolor.colored("START: Attempting to find references to pinned messages", "white", "on_blue"))
+    pagination_cursor = ''
+    channel_list = dict()
+    try:
```

Comment:
I would move this block of code out to another function so that the code is more re-usable.

Reply:
Thank you @NeuroWinter - I'll merge this then we'll implement your suggestion in the next PR! 👍